src/sentry/api/endpoints/organization_metrics.py

```diff
@@ -153,12 +153,6 @@ class OrganizationMetricsDataEndpoint(OrganizationEndpoint):
     default_per_page = 50
 
     def get(self, request: Request, organization) -> Response:
-        if not (
-            features.has("organizations:metrics", organization, actor=request.user)
-            or features.has("organizations:dashboards-releases", organization, actor=request.user)
-        ):
-            return Response(status=404)
```

Comment:
Ah, I know I just approved, but do we still need to check for `features.has("organizations:metrics", organization, actor=request.user)` here?

Reply:
leaving `organizations:metrics` here will basically 404 all the requests coming in from dashboards, so we'll need to remove it, confirmed with @matejminar / telemetry that this should be ok